alts/src/main/java/io/grpc/alts/GoogleDefaultChannelCredentials.java

```diff
@@ -44,30 +44,54 @@ private GoogleDefaultChannelCredentials() {}
    * as fallback.
    */
   public static ChannelCredentials create() {
-    ChannelCredentials nettyCredentials =
-        InternalNettyChannelCredentials.create(createClientFactory());
-    CallCredentials callCredentials;
-    try {
-      callCredentials = MoreCallCredentials.from(GoogleCredentials.getApplicationDefault());
-    } catch (IOException e) {
-      callCredentials = new FailingCallCredentials(
-          Status.UNAUTHENTICATED
-              .withDescription("Failed to get Google default credentials")
-              .withCause(e));
-    }
-    return CompositeChannelCredentials.create(nettyCredentials, callCredentials);
+    return newBuilder().build();
+  }
+
+  public static Builder newBuilder() {
+    return new Builder();
   }
 
-  private static InternalProtocolNegotiator.ClientFactory createClientFactory() {
-    SslContext sslContext;
-    try {
-      sslContext = GrpcSslContexts.forClient().build();
-    } catch (SSLException e) {
-      throw new RuntimeException(e);
+  /** Builder for {@link GoogleDefaultChannelCredentials} instances. */
+  public static final class Builder {
+    private CallCredentials callCredentials;
+
+    /** Construct GoogleDefaultChannelCredentials with a given call credential. */
+    public Builder setCallCredentials(CallCredentials callCreds) {
```

Comment:
Don't include "set". I see that one test method in AltsChannelCredentials uses set, but another one doesn't. TlsChannelCredentials.Builder and ManagedChannelBuilder don't include 'set', so let's do the same here. `callCredentials`.

Reply:
Done